src/widgets/vitals/vital-record.test.tsx

```diff
@@ -71,21 +71,21 @@ describe("<VitalRecord />", () => {
 
     await wait(() => {
       expect(wrapper).toBeDefined();
```

Comment:
If you haven't read it, I *highly* recommend [Common Mistakes with React Testing Library](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library)

Reply:
Actually, right here it makes the difference between a good failure message and a useless one. Before, the error message it produced was something like "timed out before condition met" -- it could have been any of those problems. Now it tells you which condition failed.

Reply:
Yeah, that might be fun! Though honestly everything I know and would have to share is just stuff I've read on Kent C Dodds's blog.

Reply:
Yeah that's what I would call a cosmetic improvement. The functional one (test is green or red) is not impacted (yet). Nevertheless, as already written - its better, even though I'd go with correct typings and an intrinsic correctness before excessive testing any time.

Reply:
> The functional one (test is green or red) is not impacted (yet).

Not sure I understand. Maybe it would clarify to say what you would change about this test?

> cosmetic improvement

Sure, but I think the importance of good error messages is often overlooked. 80% of coding is debugging. Having high-quality error messages is a serious quality of life issue for a programmer.

Reply:
> Not sure I understand. Maybe it would clarify to say what you would change about this test?

What I'm saying is that a test is either OK (green) or broken (red). This is the purpose of the test. When this behavior did not change (I guess it did not) the functional behavior remains unchanged. At the end, this is what matters most.

> Sure, but I think the importance of good error messages is often overlooked. 80% of coding is debugging. Having high-quality error messages is a serious quality of life issue for a programmer.

I think that is undeniably true (well, the 80% we can argue - but let's say a large fraction of dev certainly is), but if there is an issue with the test (i.e., test is now red) you'll need to start debugging anyway. Nevertheless, having a good (or more appropriate) message certainly helps, so again - this is appreciated. :+1: